assets/src/styles/base/_variables.scss

```diff
@@ -1,4 +1,5 @@
 // Breakpoints
-$mq-lg-up: "( min-width: 1200px )";
+$mq-lg-up: "( min-width: 1300px )";
+$mq-lg-down: "( max-width: 1300px )";
```

Comment:
I don't think its actually used. I'll remove for now - should encourage mobile first anyway.